test/quicer_test_lib.erl

```diff
@@ -87,6 +89,16 @@ create_file(Filename, Fmt, Args) ->
   end,
   ok.
 
+receive_all() ->
+  receive_all([]).
+
+receive_all(Res)->
+  receive
+    X ->
+      receive_all([X|Res])
+  after 0 ->
+      Res
```

Comment:
Tiny nit: maybe reverse the list to keep it in chronological order?

Reply:
that will be helpful indeed.